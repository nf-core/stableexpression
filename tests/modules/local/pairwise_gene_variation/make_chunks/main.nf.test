nextflow_process {

    name "Test Process MAKE_CHUNKS"
    script "modules/local/pairwise_gene_variation/make_chunks/main.nf"
    process "MAKE_CHUNKS"
    tag "make_chunks"
    tag "module"

    test("Should run without failures") {

        when {
            process {
                """
                ch_counts = Channel.fromPath( '$projectDir/tests/test_data/pairwise_gene_variation/make_chunks/input/counts.parquet', checkIfExists: true)
                input[0] = ch_counts
                """
            }
        }

        then {
            assert process.success
            assert snapshot(process.out).match()
        }

    }

}
