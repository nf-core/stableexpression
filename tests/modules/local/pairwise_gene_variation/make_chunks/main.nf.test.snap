{
    "Should run without failures": {
        "content": [
            {
                "0": [
                    [
                        "count_chunk.0.parquet:md5,77cbba7f85bb646d4843d756c7ba701d",
                        "count_chunk.1.parquet:md5,33d252ec9988e951a3b9dbe30d6c2e3d",
                        "count_chunk.2.parquet:md5,1a80a3d72cd2b090d1b67aa561409f7c",
                        "count_chunk.3.parquet:md5,34890e85d64bbc623477299cff4b3c24"
                    ]
                ],
                "1": [
                    [
                        "MAKE_CHUNKS",
                        "python",
                        "3.12.8"
                    ]
                ],
                "2": [
                    [
                        "MAKE_CHUNKS",
                        "polars",
                        "1.17.1"
                    ]
                ],
                "chunks": [
                    [
                        "count_chunk.0.parquet:md5,77cbba7f85bb646d4843d756c7ba701d",
                        "count_chunk.1.parquet:md5,33d252ec9988e951a3b9dbe30d6c2e3d",
                        "count_chunk.2.parquet:md5,1a80a3d72cd2b090d1b67aa561409f7c",
                        "count_chunk.3.parquet:md5,34890e85d64bbc623477299cff4b3c24"
                    ]
                ]
            }
        ],
        "meta": {
            "nf-test": "0.9.2",
            "nextflow": "24.10.3"
        },
        "timestamp": "2025-01-29T19:24:23.507551112"
    }
}